packages/components/src/organisms/Filter/FilterSlider.tsx

```diff
@@ -0,0 +1,90 @@
+import {
+  Button,
+  ButtonProps,
+  SlideOver,
+  SlideOverHeader,
+  useFadeEffect,
+  useUI,
+} from '../../'
+
+import React, { PropsWithChildren } from 'react'
+import { SlideOverDirection, SlideOverWidthSize } from '../SlideOver'
+
+export interface FilterSliderProps {
+  /**
+   * ID to find this component in testing tools (e.g.: cypress,
+   * testing-library, and jest).
```

Comment:
is `title` prop description missing?  Maybe you forgot to update this description

Reply:
Woww, Nice catch. done [here](https://github.com/vtex/faststore/pull/1687/commits/e422794e19b3ffe9df5cc6f63c683dae09f5557e).